pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -159,6 +159,9 @@ func (o *EditOptions) Run() error {
 	edit := NewDefaultEditor(o.f.EditorEnvs())
 	// editFn is invoked for each edit session (once with a list for normal edit, once for each individual resource in a edit-on-create invocation)
 	editFn := func(infos []*resource.Info) error {
+		if infos == nil || len(infos) == 0 {
```

Comment:
I'd rather do a length check prior to invoking editFn in `case NormalEditMode:` below